zipkin-ui/js/component_ui/traces.js

```diff
@@ -29,8 +28,6 @@ export default component(function traces() {
       this.timestamp = parseInt($this.data('timestamp'), 10);
       this.percentage = parseInt($this.data('servicePercentage'), 10);
     });
-    this.on(document, 'uiAddServiceNameFilter', this.addFilter);
```

Comment:
`addFilter` and `removeFilter` are removed when filter features are removed.
So this process does not finish successfully.
This process seems to be called in https://github.com/openzipkin/zipkin/blob/2e32676489acd3c807cfcc57a6e0e456a00fd607/zipkin-ui/js/page/default.js#L89 , so later processes of this line will not be executed.
So https://github.com/openzipkin/zipkin/blob/2e32676489acd3c807cfcc57a6e0e456a00fd607/zipkin-ui/js/page/default.js#L92 is not called, and then https://github.com/openzipkin/zipkin/blob/2e32676489acd3c807cfcc57a6e0e456a00fd607/zipkin-ui/js/component_ui/backToTop.js#L11 is not called, too.